tests/test-cmd/test.run.js

```diff
@@ -10,7 +10,7 @@ import {onlyInstancesOf, WebExtError, RemoteTempInstallNotSupported}
 import run, {
   defaultFirefoxClient, defaultWatcherCreator, defaultReloadStrategy,
 } from '../../src/cmd/run';
-import * as firefox from '../../src/firefox';
+import * as firefoxApp from '../../src/firefox';
```

Comment:
@rpl, sorry if this is a stupid question, but won't I have to rename all occurrences is `test.run.js`? Since it is being used as a type in `run.js`, I changed it in a few places & it works, but the same isn't true for the test file because as far as I understood, it is being used there directly & changing one occurrence breaks a lot of tests.